pkg/controller/apikey/create.go

```diff
@@ -54,12 +51,13 @@ func (c *Controller) HandleCreate() http.Handler {
 		}
 
 		if _, err := c.db.CreateAuthorizedApp(realm.ID, form.Name, form.Type); err != nil {
+			c.logger.Errorw("failed to create authorized app", "error", err)
 			flash.Error("Failed to create API key: %v", err)
 			http.Redirect(w, r, "/apikeys", http.StatusSeeOther)
 			return
 		}
 
-		flash.Alert("Created API Key for %q", form.Name)
+		flash.Alert("Successfully created API Key for %v", form.Name)
```

Comment:
Why v? Quoted seems like it would be better?

Reply:
Just trying to be consistent with our other flash messages. We can change them all to be q or v, but I figured the diff was already pretty big 😄

Reply:
Ah fair.